### DISCUSSION TITLE
Adding additional columns to intel file

### DISCUSSION THREAD

**Discussion by SliuzasLukas** [zeek/zeek#3983] [Help]
I am currently working with Security Onion, and I'm encountering an issue with our old intel file (originally from Bro) that had the columns for `meta.desc and meta.cif_confidence`. However, in our current setup, these fields are missing from the intel.log file.

I was wondering if anyone could guide me on how to include these fields again. Any insights or recommendations on how to modify the intel framework to support these fields would be greatly appreciated!

Thanks in advance for your help!

**Comment by JustinAzoff** [comment]
Hi,

add

```
@load integration/collective-intel
```

to your local.zeek, that should bring the fields back.

**Reply by SliuzasLukas** [reply]
Do you mean to intel's local.zeek file? Or to main zeek local.zeek?

**Comment by SliuzasLukas** [comment]
Hi,

I have tried this and it's working 90%. For some reason meta.desc isn't shown in the log
